cachi2/core/package_managers/yarn/main.py

```diff
@@ -5,6 +5,7 @@
 from cachi2.core.models.output import Component, EnvironmentVariable, RequestOutput
 from cachi2.core.package_managers.yarn.project import (
     Project,
```

Comment:
nitpick: typo in commit message - plugins: set to an empty array

Reply:
Fixed.